msbuild-api-docs/xml/Microsoft.Build.BuildEngine/Project.xml

```diff
@@ -1121,7 +1121,7 @@
       </ReturnValue>
       <Docs>
         <summary>Gets a collection of the item groups specified with the ItemGroup element in the project.</summary>
-        <value>A <see cref="T:Microsoft.Build.BuildEngine.BuildItemGroupCollection" /> containing the item groups specified in the project.</value>
+        <value>A <see cref="T:Microsoft.Build.BuildEngine.BuildItemGroupCollection" /> containing the item groups specifiied in the project.</value>
```

Comment:
Can you run it again without the XML files? It looks like most/all of the documentation changes are undesirable.

Reply:
The XML for 4.8.1 has been removed and the CI job is being re-run.

Reply:
Sorry @ghogen, this will reset some of your changes.